docs/resources/postgres_hba_conf.md.erb

```diff
@@ -0,0 +1,103 @@
+---
+title: About the postgres_hba_conf Resource
+---
+
+# postgres_hba_conf
+
+Use the `postgres_hba_conf` InSpec audit resource to test the client authentication data defined in the pg_hba.conf file.
+## Syntax
+
+An `postgres_hba_conf` InSpec audit resource block declares client authentication data that should be tested:
+
+    describe postgres_hba_conf.where { pg_username == 'filter_value' } do
```

Comment:
Should we use `pg_username` here? Shouldn't we use the more generic version here as you use in code?
```
describe postgres_hba_conf.where { type == 'local' } do
  its('auth_method') { should eq ['peer'] }
end
```